client/components/editor/ActivityContent/index.vue

```diff
@@ -170,6 +172,12 @@ export default {
       [[removeSelection, false], [isSelected, true]].forEach(([items, isSelected]) => {
         items.forEach(({ elementId, ...user }) => this.selectElement(elementId, user, isSelected));
       });
+    },
+    $route(route) {
+      const { elementId } = route.query;
+      if (!elementId) return;
+      this.selectElement(elementId);
```

Comment:
We should somehow reuse this section within isLoading watcher and document when this is being used.